projects/packages/blaze/src/class-dashboard-rest-controller.php

```diff
@@ -163,6 +163,17 @@ public function register_rest_routes() {
 			)
 		);
 
+		// WordAds DSP API Site Stats routes
+		register_rest_route(
+			static::$namespace,
+			sprintf( '/sites/%1$d/wordads/dsp/api/v1.1/sites/%1$d/stats(?P<sub_path>[a-zA-Z0-9-_\/]*)(\?.*)?', $site_id ),
+			array(
+				'methods'             => WP_REST_Server::READABLE,
```

Comment:
We will also need the EDITABLE version of this controller. I am working on the export stats endpoint, and I will need to use the POST request there.

I think its fine to use the campaigns as an example here:
https://github.com/Automattic/jetpack/blob/f676e1cfddd309a841adafe1e2b958c21f0420da/projects/packages/blaze/src/class-dashboard-rest-controller.php#L136-L153